src/main/java/com/bootcamp/demo/FirebaseController.java

```diff
@@ -47,6 +48,11 @@ private void initFirestore() throws IOException {
         firestoreDB = FirestoreClient.getFirestore();
     }
 
+    @Bean
```

Comment:
The `@Bean` annotation can be used only with `@Configuration` annotated classes. Its purpose is to create a new Bean. 
Pull Request https://github.com/remus-lupin-team/payments-jb/pull/15 has a good example of a Firestore Repository/DAO. You could build on that implementation as soon as it's merged with master.

Reply:
Updated base branch to feature/payment-by-card instead of master. Those are not my modifications